src/middleware/mapper.js

```diff
@@ -4,12 +4,27 @@ const objectMapper = require('object-mapper')
 
 const logger = require('../logger')
 
-const transformInput = mappingSchema => ctx => {
-  logger.debug(`Validation Schema: ${JSON.stringify(ctx.validation)}`)
+const createMappedObject = (ctx, mappingSchema) => {
+  if (!mappingSchema || mappingSchema == undefined) {
```

Comment:
undefined is falsy as well. undefined and null are both equivalent to false. it should be 
 `if (!mappingSchema)`